node/src/components/gossiper/tests.rs

```diff
@@ -129,7 +129,12 @@ impl reactor::Reactor for Reactor {
         let storage = Storage::new(WithDir::new(storage_tempdir.path(), storage_config)).unwrap();
 
         let deploy_acceptor = DeployAcceptor::new();
-        let deploy_gossiper = Gossiper::new_for_partial_items(config, get_deploy_from_storage);
+        let deploy_gossiper = Gossiper::new_for_partial_items(
+            "deploy_gossiper",
```

Comment:
We shouldn't be needing metrics in tests :/

Reply:
Is there any harm in having them? The metrics are a part of the component now.

Reply:
they are not used, make test slower etc.

Reply:
They make tests slower by a single update of an atomic integer value, which is a few (one?) instruction. On the other hand, we have a lot less complexity and get to test a little bit closer to the "real world".

Reply:
I doubt we will create that many events in the unit tests that those metrics become useful.

> They make tests slower by a single update of an atomic integer value, which is a few (one?) instruction.

IIRC there's multiple atomic integers.